var/spack/repos/builtin/packages/mesa-glu/register-long.patch

```diff
@@ -0,0 +1,28 @@
+From e5b42317078a1dbbf7ef9f1431ef6dcb711c7026 Mon Sep 17 00:00:00 2001
+From: Aiden Grossman <agrossman154@yahoo.com>
+Date: Mon, 10 Jul 2023 17:59:18 -0700
+Subject: [PATCH] Change register long to long to fix compile error
+
+When using -std=c++17, using register long will throw an error. This
+patch switches all instances of register long to long to fix this.
```

Comment:
Can you add these two lines as a comment above the patch directive in the `package.py` file?